### DISCUSSION TITLE
Min jail time

### DISCUSSION THREAD

**Discussion by bobhenl** [Zrips/CMI#7906] [Enhancement]
Hi, I'd like to ask for a minimum jail time option in config (or via permission), cause some staff members abuse the jail cmd by jailing for 1s or so to get an advantage in combat. Yeah, I can solve it by kicking them from the team, but some prevention like permission or minimum time set will be nice, as well.

**Comment by Zrips** [comment]
First of all, staff like that should not be a staff, IMO. Other than this, min time is quite deprecated limitation, if someone wants to abuse jail command than they will abuse it either way, jailing for a second or for 10 minutes, still same abuse, difference is that in second instance player who gets jailed will be more annoyed by being in jail for extended period of time which can result in worse results than just jailing for a second or two. 
Don't get me wrong here, having permission like that could play some role, but trying to avoid in additing not necessary permission nodes as we have quite few of those already.
